src/pages/help-centre.js

```diff
@@ -1,13 +1,688 @@
+            },
+            {
+                title: localize('Changing your personal details'),
+                category: localize('Account'),
+                sub_category: localize('Managing your account'),
+                label: 'changing-your-personal-details',
+                content: <ChangingPersonalDetails />,
+            },
+            {
+                title: localize('Recovering your password'),
+                category: localize('Account'),
+                sub_category: localize('Managing your account'),
+                label: 'recovering-your-password',
+                content: <RecoveringYourPassword />,
+            },
+            {
+                title: localize('Authenticating your account'),
+                category: localize('Account'),
+                sub_category: localize('Managing your account'),
+                label: 'recovering-your-password',
+                content: <AuthenticatingYourAccount />,
+            },
+        ],
+    },
+    {
+        category: localize('Deposits and withdrawals'),
+        articles: [
+            {
+                title: localize('Making a deposit'),
+                category: localize('Deposits and withdrawals'),
+                sub_category: localize('Deposits'),
+                label: 'making-a-deposit',
+                content: <MakingADeposit />,
+            },
+            {
+                title: localize('Expired verification link'),
+                category: localize('Deposits and withdrawals'),
+                sub_category: localize('Deposits'),
+                label: 'expired-verification-link',
+                content: <ExpiredVerificationLink />,
+            },
+            {
+                title: localize('Withdrawal processing time'),
+                category: localize('Deposits and withdrawals'),
+                sub_category: localize('Withdrawals'),
+                label: 'withdrawal-processing-time',
+                content: <WithdrawalProcessingTime />,
+            },
+        ],
+    },
+]
+
+const getRelatedArticles = (article_list, selected_article) =>
+    article_list.filter(
+        article =>
+            article.sub_category === selected_article.sub_category &&
+            article.title !== selected_article.title,
+    )
+
+const getAllArticles = articles =>
+    articles
+        .map(category => category.articles)
+        // flatten the array, gatsby build does not support .flat() yet
+        .reduce((arr, article_arr) => arr.concat(article_arr), [])
+
+const SearchSection = styled.section`
+    ${Backdrop} {
+        max-height: ${props => (props.show ? '1000px' : '0')};
+        transition: ${props =>
+            props.has_transition ? 'max-height 0.6s ease-in-out' : 'none'};
+        overflow: hidden;
+    }
+`
+const Backdrop = styled.div`
+    background-color: var(--color-black);
+    padding: 12rem 0;
+`
+
+const SearchIconBig = styled(SearchIcon)`
+    position: absolute;
+    left: 0;
+    top: 3px;
```

Comment:
can we use pixel?

Reply:
For fixed values we should use pixels yes (if it's not supposed to scale across screen sizes)